rebasehelper/cli.py

```diff
@@ -207,6 +213,13 @@ def build_parser():
                  "and note that = before it is mandatory"
         )
         parser.add_argument(
+            "--srpm-builder-options",
+            default=None,
+            metavar="SRPM_OPTIONS",
```

Comment:
This should also be `SRPM_BUILDER_OPTIONS`, and the same in the help text below.